### PR TITLE
Add SiP Link

### LINKED ISSUE #2801: add link to SiP to Educator Page
Add a sentence on the educator page so that we have a link to Scratch in Practice. 
This is the sentence with the link to add: 
[**Scratch in Practice**](http://sip.scratch.mit.edu) shares ideas and resources from Scratch Team and educators around the world, with a new theme each month. 

Add that sentence directly before the current sentence: "Educator Guides show you how to prepare and run Scratch classes and workshops."

### PR DESCRIPTION
**Pull request by apple502j**
### Resolves:
Resolves #2801 

### Changes:
Adds SiP link to educator landing

### CODE REVIEW

**Comment by benjiwheeler** [comment]
Blocked while we check on formatting with @ntlrsk

**Comment by benjiwheeler** [comment]
Blocked while waiting for new language from ScratchEd team

**Comment by benjiwheeler** [comment]
formatting, new language are good; now waiting on SiP icon design

**Comment by benjiwheeler** [comment]
Current appearance:

image

**Comment by ntlrsk** [comment]
Wondering if it can add (SiP) after Scratch in Practice so that it reads as a name of something (and to introduce that abbreviation)? Otherwise looks great to me!

**Comment by benjiwheeler** [comment]
Updated:

image

**Comment by ntlrsk** [comment]
Great to see. Thank you! 👍

File: src/_colors.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /* UI Primary Colors */
 $ui-blue: hsla(215, 100, 65, 1); // #4C97FF Motion Primary
 $ui-blue-dark: hsla(215, 65, 55, 1); // #3373CC Motion Secondary
+$ui-blue-very-light:  hsla(215, 100, 95, 1);
```
Comment (carljbowman) [inline src/_colors.scss]: @benjiwheeler - Generally we try not to add color variables, we already have an excess. But that's on me, I should have called out that the background of the container should be a transparent blue: `$ui-blue-10percent`

**Comment by benjiwheeler** [comment]
Responsive layouts:

at 450px wide:
image

at 850 px wide:
image

File: src/_colors.scss (hunk 1 above)
Comment (benjiwheeler) [reply src/_colors.scss]: The problem is the transparency -- behind it is purple! But I'll see if I can replace that with white.

**Comment by benjiwheeler** [comment]
ok, made it work with the existing blue10percent!

image

**Comment by benjiwheeler** [comment]
Final version!

image

**Comment by apple502j** [comment]
lol, ST members, I've never seen this kind of strange PR, maybe April Fools's joke?

**Comment by benjiwheeler** [comment]
@apple502j Not sure what you mean! You mean how much it expanded in scope from the original version?

Thanks for submitting it, by the way, it ended up kicking off a much bigger discussion about how to balance the focus on the page!

**Pull request merged**